### PR TITLE
Flask error handling

### PR DESCRIPTION
**Pull request by Loonride**
Use flask app errorhandler to handle exceptions coming from routes (Fixes https://github.com/funcx-faas/funcx-web-service/issues/239)

Clubhouse story: https://app.clubhouse.io/funcx/story/9911/use-flask-app-errorhandler-in-web-service

Also related to the JSON logging effort as we want to distinguish errors from normal HTTP responses

### CODE REVIEW

**Comment by Loonride** [comment]
[ch9911]

File: funcx_web_service/routes/funcx.py [hunk 1]
```diff
@@ -655,10 +640,10 @@ def get_ep_stats(user: User, endpoint_id):
 
     try:
         if not authorize_endpoint(user_id, endpoint_id, None, token):
-            return create_error_response(EndpointAccessForbidden(endpoint_id), jsonify_response=True)
+            raise EndpointAccessForbidden(endpoint_id)
     except Exception as e:
         # could be EndpointNotFound
-        return create_error_response(e, jsonify_response=True)
+        raise e
```
Comment (joshbryan-globus) [inline funcx_web_service/routes/funcx.py:646]: Is there value in catching and immediately re-raising?

File: funcx_web_service/routes/funcx.py [hunk 2]
```diff
@@ -248,10 +227,16 @@ def submit(user: User):
     final_http_status = 200
     success_count = 0
     for task in tasks:
-        res = auth_and_launch(
-            user_id, function_uuid=task[0], endpoint_uuid=task[1],
-            input_data=task[2], app=app, token=token, task_group_id=task_group_id, serialize=serialize
-        )
+        task_uuid = str(uuid.uuid4())
+        try:
+            res = auth_and_launch(
```
Comment (joshbryan-globus) [inline funcx_web_service/routes/funcx.py:232]: `auth_and_launch` is also used in the [automate api](https://github.com/funcx-faas/funcx-web-service/blob/main/funcx_web_service/routes/automate.py#L72).  Given the signature changed to raise an exception should that code also change?

**Review by joshbryan-globus — APPROVED** [review]
Two minor comments/questions.  Otherwise LGTM.

File: funcx_web_service/routes/funcx.py (hunk 2 above)
Comment (Loonride) [reply funcx_web_service/routes/funcx.py]: Good catch, I instead changed the signature back to the original and moved the try block into the method itself so we don't have to wrap it outside

File: funcx_web_service/routes/funcx.py (hunk 1 above)
Comment (Loonride) [reply funcx_web_service/routes/funcx.py]: Nope, removed this.

**Comment by Loonride** [comment]
@joshbryan-globus Ready for review again

**Review by joshbryan-globus — APPROVED** [review]

**Pull request merged**